src/database/models/index.js

```diff
@@ -0,0 +1,35 @@
+const fs = require('fs');
+const path = require('path');
+const Sequelize = require('sequelize');
+const envConfigs = require('../config/config');
+
+const basename = path.basename(__filename);
+const env = process.env.NODE_ENV || 'development';
```

Comment:
Is there any particular reason this is using `require` ?

Reply:
Because the models are created automatically, they come in es5